apps/app/src/containers/Layout.tsx

```diff
@@ -23,8 +23,8 @@ export const Main = forwardRef<HTMLElement, { children: React.ReactNode }>(
 export const Layout = (props: { children: React.ReactNode }) => {
   const fullSize = useMatch("/:accountSlug/:projectName");
   return (
-    <div className={clsx(fullSize && "h-screen", "min-h-screen")}>
-      <header>
+    <div className={clsx(fullSize && "h-screen", "flex min-h-full flex-col")}>
+      <header className="flex-shrink-0">
```

Comment:
Layout was broken that's why I made the change. I will fix it...

Reply:
If you revert it, then there is lot of other `shrink-0` to add.

Reply:
I removed it on sub nav and tabs I think. With the "legacy" layout you are puting back, every elements in the page should have `shrink-0`. Else they will be "compacted" if there is a big list.

Anyway I removed it for a reason and puting it back is not the good solution...